src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -1056,81 +1134,165 @@ class Simplex_tree {
     return insert_simplex_raw(copy, filtration);
   }
 
-    /** \brief Insert a N-simplex and all his subfaces, from a N-simplex represented by a range of
+  /**
+   * @brief List of insertion strategies of a simplex \f$ \sigma \f$.
+   */
+  enum class Insertion_strategies {
```

Comment:
Is it a good idea to make it plural?

Reply:
I don't know, is it better in singular? I just made it plural because it is an enumeration with more than one element. I can make it singular if that is the convention.

Reply:
The standard uses for instance `https://en.cppreference.com/w/cpp/types/numeric_limits/float_round_style` singular. I don't know if we already have enums in Gudhi and what conventions they follow.

Reply:
There is one with `Extended_simplex_type`, so I will make it singular.